server/interfaces/prisonApi/agencies.ts

```diff
@@ -0,0 +1,54 @@
+export interface Agencies {
```

Comment:
A bit confusing to have these interfaces with plural names when they are just one thing.